src/applications/facility-locator/actions/index.js

```diff
@@ -127,17 +187,50 @@ export const fetchLocations = async (
   center,
   radius,
 ) => {
+  let data = {};
+
   try {
-    const data = await LocatorApi.searchWithBounds(
-      address,
-      bounds,
-      locationType,
-      serviceType,
-      page,
-      center,
-      radius,
-    );
-    // Record event as soon as API return results
+    if (
+      locationType === LocationType.URGENT_CARE &&
+      (!serviceType || serviceType === Object.keys(urgentCareServices)[0])
```

Comment:
Again I'd rather hard code `AllUrgentCare` here rather than relying on the order of the keys.